app/controllers/api/v1/athletes_controller.rb

```diff
@@ -9,7 +9,13 @@ def show
   end
 
   def update
-    render json: { status: 'Thank you for casting your vote!' } if @result.updated_votes(params)
+    if params[:vote]
+      @result.updated_votes(params)
+      render json: { status: 'Thank you for casting your vote!' }
+    else
+      @athlete.update(athlete_params)
+      render json: { status: 'Athlete updated successfully!' }
+    end
```

Comment:
Didn't think of it yesterday but you guys could make use of the edit action instead of update. That way it would be cleaner and you could get ride of the if - statment here. Please change this